test/uk/gov/hmrc/nisp/controllers/AccountControllerSpec.scala

```diff
@@ -199,7 +199,7 @@ class AccountControllerSpec extends UnitSpec with MockitoSugar with BeforeAndAft
 
       "return COPE page for contracted out (B) user" in {
         val result = MockAccountController.showCope()(authenticatedFakeRequest(mockUserIdContractedOut))
-        contentAsString(result) should include ("You were contracted out ")
+        contentAsString(result) should include ("Your COPE estimate is")
```

Comment:
How can the text have changed unless one of your commits did that, is this not a broken test?